packages/ejson/package.js

```diff
@@ -1,10 +1,10 @@
 Package.describe({
   summary: "Extended and Extensible JSON library",
-  version: '1.0.13'
+  version: '1.0.14'
 });
 
 Package.onUse(function (api) {
-  api.use(['underscore', 'base64']);
+  api.use(['ecmascript', 'underscore', 'base64']);
```

Comment:
That's a good question - the changes I put in place (replacing some `underscore` function calls) were just intended to address the issue in #5175. Underscore is still used fairly heavily in the rest of the `ejson` package - here are all of the unique `underscore` calls:

```
isNaN
has
isEmpty
any
all
isArray
isArguments
each
size
```

I'm happy to remove `underscore` completely, but I'd just be replicating a lot of the libraries code to address the above functionality (excluding the obvious ones like `each`). With the upcoming `underscore` to `lodash` switch (https://github.com/meteor/meteor-feature-requests/issues/48), completely removing `underscore` from `ejson` might not be needed (yet). Again, happy to switch it out if you say make it happen, so let me know. Thanks!